packages/office-ui-fabric-react/src/common/_semanticSlots.scss

```diff
@@ -40,14 +40,14 @@ $buttonBackgroundHoveredColor: '[theme:buttonBackgroundHovered, default: #edebe9
 $buttonBackgroundCheckedHoveredColor: '[theme:buttonBackgroundCheckedHovered, default: #edebe9]'; 
 $buttonBackgroundPressedColor: '[theme:buttonBackgroundPressed, default: #edebe9]'; 
 $buttonBackgroundDisabledColor: '[theme:buttonBackgroundDisabled, default: #f3f2f1]'; 
-$buttonBorderColor: '[theme:buttonBorder, default: transparent]'; 
```

Comment:
wait why did this change?

Reply:
oh nm this is the 7.0 branch

Reply:
Yeah. This was something that was not picked up with the first change of the neutral colors due to you changing at the same time the script that auto-generates these files. Now it's actually picking it up.